hfs-nightly-charges/development/main.tf

```diff
@@ -0,0 +1,136 @@
+locals {
+  ssm_path = "${var.housing_finance_prefix}/${var.environment}"
+}
+
+# Environment variables for the Task
+data "aws_ssm_parameter" "housing_finance_mssql_dbhost" {
+  name = "${local.ssm_path}/db-host"
+}
+data "aws_ssm_parameter" "housing_finance_mssql_database" {
+  name = "${local.ssm_path}/mssql-database"
+}
+data "aws_ssm_parameter" "housing_finance_mssql_username" {
+  name = "${local.ssm_path}/mssql-username"
+}
+data "aws_ssm_parameter" "housing_finance_mssql_password" {
+  name = "${local.ssm_path}/db-password"
+}
+data "aws_ssm_parameter" "google_api_key" {
+  name = "${local.ssm_path}/google-application-credentials-json"
+}
+data "aws_ssm_parameter" "charges_batch_years" {
+  name = "${local.ssm_path}/charges-batch-years"
+}
+data "aws_ssm_parameter" "batch_size" {
+  name = "${local.ssm_path}/bulk-insert-batch-size"
+}
```

Comment:
The bundling of these env vars with the environment was removed in commit [6a5e1786](https://github.com/LBHackney-IT/mtfh-finance-infrastructure/pull/19/commits/6a5e17866f4de2d30ef6d8242a2cd4c86c7e3e0b). So we can probably get rid of these if they're not used anywhere else.